engine/execution/computation/computer/computer.go

```diff
@@ -255,6 +257,7 @@ func (e *blockComputer) getRootSpanAndCollections(
 	collections = append(
 		collections,
 		collectionItem{
+			blockId:         blockId,
```

Comment:
integration test picked up the fact that i forgot to initialized the field here.  I'll added some unittest to verify the chunk entries as well.